### PR TITLE
MMT-1647 Updating MMT to add GTM html code

### PR DESCRIPTION
**Pull request by rosycordova**
Update html to add Google Tag Manager (GTM) code to head and body tags.

### CODE REVIEW

File: config/environments/sit.rb [hunk 1]
```diff
@@ -99,6 +99,7 @@
   config.cmr_env = 'sit'
   config.echo_env = 'sit'
   config.analytics_id = 'UA-62340125-5'
+  config.tag_manager_id = 'GTM-TESTTEST'
```
Comment (jceaser) [inline config/environments/sit.rb]: what was the logic for not using the ID for SIT and UAT? I see why to not use it for development and test. Not yet suggesting a change, just want to ask what lead us to here.

File: app/views/shared/_google_tag_manager_body.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<!-- Google Tag Manager (noscript) -->
+<noscript>
+<iframe src="https://www.googletagmanager.com/ns.html?id=<%= Rails.configuration.tag_manager_id %>"
```
Comment (charleshuang80) [inline app/views/shared/_google_tag_manager_body.html.erb]: I feel like we should wrap both the `body` and `head` partial contents within an if statement similar to how we had it for the `analytics_id`:
`<% if Rails.configuration.respond_to?(:tag_manager_id) -%>`
That way we wouldn't need to add fake ids, and not need to add them for development or test.

File: config/environments/sit.rb (hunk 1 above)
Comment (rosycordova) [reply config/environments/sit.rb]: Peggy from EMS replied that same ID can be used in all environments.  Updated code & resubmitted.

File: app/views/shared/_google_tag_manager_body.html.erb (hunk 2 above)
Comment (rosycordova) [reply app/views/shared/_google_tag_manager_body.html.erb]: added if statement & resubmitted.

**Pull request merged**